.github/workflows/update.yml

```diff
@@ -12,32 +12,45 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Get Zotero version
+      
+      - name: Detect Zotero Linux Version
         id: zotero
-        uses: oprypin/find-latest-tag@v1
-        with:
-          repository: zotero/zotero
+        run: |
+          # Use curl to follow the redirect and capture the final URL
+          url=$(curl -Ls -o /dev/null -w %{url_effective} 'https://www.zotero.org/download/client/dl?channel=release&platform=linux-x86_64')
+          
+          # Extract the version from the URL
+          version=$(echo "$url" | grep -oP 'Zotero-\K[0-9.]+(?=_linux-x86_64.tar.bz2)')
+          
+          echo "Latest Zotero version for Linux: $version"
+          echo "::set-output name=version::$version"
```

Comment:
thanks for the the pr, it went well on my machine to test the correct version for linux!

just one nitpick, it seems that github has deprecated "::set-output" 
https://github.blog/changelog/2023-07-24-github-actions-update-on-save-state-and-set-output-commands/

I'll happily merge this if you could you change this into:
`echo "version::$version" >> "$GITHUB_OUTPUT"`